### PR TITLE
Provide update timestamp of variable values

### PR DESCRIPTION
**Pull request by lobstaj**

### CODE REVIEW

File: src/pyupway/models/variablevalue.py [hunk 1]
```diff
@@ -10,3 +11,4 @@ class VariableValue:
     Value: int | float | str | bool | None
     Unit: str | None
     EnumValue: str | None
+    UpdatedAt: datetime
```
Comment (lemanjo) [inline src/pyupway/models/variablevalue.py]: Breaks MyUpway Service. I don't remember if the update timestamp is available there. If it is, we should include it also.
If it's not available should we do:
`UpdatedAt: datetime | None`
and then make sure the class instantiation works in myupwayservice.py

**Review by lemanjo — CHANGES_REQUESTED** [review]
Make sure it doesn't break the myupwayservice

File: src/pyupway/models/variablevalue.py (hunk 1 above)
Comment (lobstaj) [reply src/pyupway/models/variablevalue.py]: Yes, that's a good point. Unfortunately I don't think I have access to myupway anymore, so I cannot really offer more than the `| None`.

Was there anything else needed to make sure the class instantiation works in MyUpwayService ?

Comment (lemanjo) [reply src/pyupway/models/variablevalue.py]: I think this should be changed to strict assignments, just to be safe.
`results.append(VariableValue(id, name, enumerator, value, unit))`
https://github.com/lemanjo/pyupway/blob/master/src/pyupway/services/myupwayservice.py#L142

Comment (lobstaj) [reply src/pyupway/models/variablevalue.py]: Added a separate commit which does that (and also harmonizes the parameter order).

**Comment by lemanjo** [comment]
Awesome. Thank you!

**Pull request merged**